main.go

```diff
@@ -208,6 +209,19 @@ func securityHandler(h http.Handler) http.Handler {
 	})
 }
 
+// healthcheckHandler uses the provided handler for /health endpoint, and serves any other traffic to the next handler in chain
+func healthcheckHandler(hc func(w http.ResponseWriter, req *http.Request)) func(h http.Handler) http.Handler {
```

Comment:
We should consider adding this function to one of the libraries - dp-healthcheck?

Reply:
good point! I think it would be very useful